backend/service/k8s/clientset.go

```diff
@@ -64,10 +70,29 @@ func newClientsetManager(rules *clientcmd.ClientConfigLoadingRules) (ClientsetMa
 		if err != nil {
 			return nil, err
 		}
-
 		lookup[name] = &ctxClientsetImpl{Interface: clientset, namespace: ns, cluster: ctxInfo.Cluster}
 	}
 
+	// If there is no configured cluster produced fallback to InClusterConfig
+	if len(lookup) == 0 {
+		logger.Warn("no kubeconfig was found, falling back to InClusterConfig")
+
+		restConfig, err := rest.InClusterConfig()
+		switch err {
+		case rest.ErrNotInCluster:
+			logger.Warn("not in a kubernetes cluster, unable to configure kube clientset")
+			lookup[inCluster] = &ctxClientsetImpl{Interface: nil, namespace: "default", cluster: inCluster}
```

Comment:
why append this broken clientset?

Reply:
The UI wont render if we have an empty lookup map

Reply:
This was purely to satisfy running clutch locally without any kubeconfig available

Reply:
FE fails to load any k8s workflows if we dont have a cluster available even if it is bogus.

```
Uncaught TypeError: Cannot read property 'stringValue' of undefined
    at hydrator.tsx:105
```
![Screen Shot 2020-07-30 at 10 52 41 AM](https://user-images.githubusercontent.com/2250844/88956843-d58a8e00-d252-11ea-9fda-9c0807237fa7.png)

Reply:
ah. that's a separate bugfix that derek is working on #147 (already merged) #149 (in progress). let's not account for it here.

Reply:
ah alright sounds good let me delete